### DISCUSSION TITLE
.Net 6 client for older oracle database

### DISCUSSION THREAD

**Discussion by kccarter76** [oracle/dotnet-db-samples#245] [Q&A] [answered]
I have been tasked with a requirement to connect to an older oracle database server. It could possible be 10g, I am waiting on confirmation. 

What version of the Oracle ODP.Net client supports interaction with 10g while supporting .Net 6?

**Comment by alexkeh** [comment, marked as answer]
10g has been out of support for about 9 years now. There is no official ODP.NET support for .NET 6 and and DB 10g. That's not to say that ODP.NET 21c doesn't work with DB 10g. We just haven't tested it since it's not officially supported anymore.

**Reply by kccarter76** [reply]
thank you for your response, we will most likely be testing that to see if that will work. if not, some very hard decisions will need to made on our end.
